src/reactviews/pages/ConnectionDialog/connectButton.tsx

```diff
@@ -31,7 +34,7 @@ export const ConnectButton = ({style}: {style?: CSSProperties}) => {
 			}
 			iconPosition="after"
 			icon={ connectionDialogContext.state.connectionStatus === ApiStatus.Loading ? <Spinner size='tiny' /> : undefined}>
-				Connect
+				{CONNECT}

```

Comment:
is there a reason to pull this out into its own const rather than just having `{l10n.t("Connect")`?